docs-src/concepts/what-is-a-retry-policy.md

```diff
@@ -91,8 +91,8 @@ Non-Retryable Errors = []
 
 - **Description**: Specifies errors that shouldn't be retried.
   - **Default is none.**
-  - If one of those errors occurs, the [Activity Task Execution](#activity-task-execution) or [Workflow Execution](#workflow-execution) is not retried.
-  - The errors are matched against the `type` field of the [ApplicationFailure](/kb/failures#application-failure).
+    - The errors are matched against the `type` field of the [ApplicationFailure](/kb/failures#application-failure).
+  - If one of those errors occurs, the [Workflow Execution](#workflow-execution) won't be retried unless Workflow Retry options are set to allow retries. [Activity Task Executions](#activity-task-execution) will retry unless `temporal.NewNonRetyableError` is returned from the Activity.
```

Comment:
Some problems here:

* This is a bit confusing. If one of those errors occurs, the thing won't be retried, regardless of whether that "thing" is workflow execution or activity task execution.
* Throwing/returning/raising an application error that is marked non-retryable won't retry the thing, regardless of whether that "thing" is workflow execution or activity task execution.
* `temporal.NewNonRetyableError` (sic, fix spelling) is a Go only thing but this is general documentation